apostrophecms/apostrophe
# PRO-2898 PRO-3018 unpin tiptap, pin vue-loader

## Summary

* Unpinned tiptap as the tiptap team has made releases that resolve the packaging errors that caused us to pin it in 3.22.1.
* Pinned `vue-loader` to the `15.9.x` minor release series for now. The `15.10.0` release breaks support for using `npm link` to develop the `apostrophe` module itself.

## What are the specific steps to test this change?

* Check out this branch of apostrophe
* `npm install`, then `npm update` (important)
* `npm link`
* Install testbed if not already installed
* `npm link apostrophe --legacy-peer-deps`
* If `npm run dev` fails on first try, `cd node_modules/apostrophe && npm install && cd ../.. && npm run dev` (unrelated, annoying issue with `npm link` command)

Admin UI should build and run normally and tiptap rich text editor should function (if the tiptap bug came back it would not even start up).

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
